.github/workflows/rebuild_native_libs.yml

```diff
@@ -58,15 +58,15 @@ jobs:
             core/src/main/resources/io/questdb/bin/darwin-aarch64/libquestdb.dylib
             core/src/main/bin/darwin-aarch64/libjemalloc.dylib
           key: nativelibs-armosx-${{ github.sha }}
-      - name: Save darwin-amd64 Libraries to Cache
+      - name: Save darwin-x86-64 Libraries to Cache
         if: ${{ matrix.os == 'macos-13' }}
         uses: actions/cache/save@v3
         with:
           path: |
-            core/src/main/resources/io/questdb/bin/darwin-amd64/libquestdb.dylib
-            core/src/main/bin/darwin-amd64/libjemalloc.dylib
+            core/src/main/resources/io/questdb/bin/darwin-x86-64/libquestdb.dylib
+            core/src/main/bin/darwin-x8664/libjemalloc.dylib
```

Comment:
Why `x8664` instead of `x86-64`?

Reply:
Resolved in a2f44894ddd9e8f83314f0dfa2bc72ad8cd73d4b.